<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertSolverDialogUi</class>
 <widget class="QDialog" name="InsertSolverDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Solver</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="insGroupBox">
     <property name="title">
      <string>New solver</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <property name="margin">
       <number>8</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Method Description:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editMethod"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Method Name:</string>
        </property>
        <property name="buddy">
         <cstring>editMethod</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>Method Path:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="editDescription"/>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="editPath"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
